themes/mongodb/src/js/componentStitchSidebar.js

```diff
@@ -0,0 +1,147 @@
+const SLIDE_ANIMATION_TIME_MS = 400;
```

Comment:
This file doesn't build due to linting issues: make sure you actually load this component in `controller.js` and test the build.

Reply:
Oops good catch. Added!